src/openforms/forms/models/form_variable.py

```diff
@@ -98,6 +105,15 @@ def create_for_formstep(self, form_step: "FormStep") -> List["FormVariable"]:
                 )
             )
 
+            if fix_prefill_config:
+                for v in form_variables:
+                    if v.prefill_plugin != "" and v.prefill_attribute == "":
```

Comment:
```suggestion
                    if v.prefill_plugin and not v.prefill_attribute:
```
we can make this a bit more readable